go.mod

```
@@ -336,4 +337,4 @@ require (
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
 
-replace github.com/cosmos/cosmos-sdk => github.com/bryanchriswhite/cosmos-sdk v0.50.1-fix-sequencer-check
+replace github.com/cosmos/cosmos-sdk => github.com/okdas/cosmos-sdk v0.50.1-fix-sequencer-check-2
```

As [discussed](https://discord.com/channels/824324475256438814/1138895490331705354/1205219131729776640), we can use theirs now.
```suggestion
replace github.com/cosmos/cosmos-sdk => github.com/rollkit/cosmos-sdk v0.50.1-rollkit-v0.11.19
```

---

@bryanchriswhite yup - this branch is on `v0.50.1-rollkit-v0.11.19-no-fraud-proofs`, did you leave the comment on an old commit? 🤔